pkgs/development/python-modules/aggdraw/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pytestCheckHook
+, python
+, pillow
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "aggdraw";
+  version = "1.3.15";
+
+  src = fetchFromGitHub {
+    owner = "pytroll";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-w3HlnsHYB0R+HZOXtzygC2RST3gllPI7SYtwSCVXhTU=";
+  };
+
+  checkInputs = [
+    numpy
+    pillow
+    pytestCheckHook
```

Comment:
```suggestion
    pytest
```

Reply:
But since we are using a custom checkPhase we are not using it.

Reply:
Oh okay, good point